### PR TITLE
Samples for message passing docs

### PR DESCRIPTION
**Pull request by dandavison**
This PR adds the code samples to accompany the Python message-passing docs (https://docs.temporal.io/develop/python/message-passing, added in https://github.com/temporalio/documentation/pull/2976)

### CODE REVIEW

File: message_passing/introduction/workflows.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        previous_language, self.language = self.language, language
+        return previous_language
+
+    @workflow.query
+    def get_language(self) -> Language:
+        return self.language
+
+
+@activity.defn
+async def call_greeting_service(to_language: Language) -> Optional[str]:
```
[the remark this answers is not in the document]
Comment (dandavison) [reply message_passing/introduction/workflows.py]: Good call, done. d0f7ef89f0c3bd1135422e66a7fb75f5587df610

File: message_passing/introduction/README.md
[the remark this answers is not in the document]
Comment (dandavison) [reply message_passing/introduction/README.md:1]: Good catch, done.

File: message_passing/safe_message_handlers/README.md
[the remark this answers is not in the document]
Comment (dandavison) [reply message_passing/safe_message_handlers/README.md:1]: Good catch, done.

File: message_passing/introduction/workflows.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import asyncio
+from dataclasses import dataclass
+from datetime import timedelta
+from typing import List, Optional
+
+from temporalio import workflow
+from temporalio.exceptions import ApplicationError
+
+from message_passing.introduction import Language
+from message_passing.introduction.activities import call_greeting_service
```
[the remark this answers is not in the document]
Comment (dandavison) [reply message_passing/introduction/workflows.py]: Thanks, done.

**Review by cretz — APPROVED** [review]

**Comment by dandavison** [comment]
Thanks @cretz

**Pull request merged**